koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -678,7 +687,23 @@ def handler(self, src, target, opts=None):
         build_tag = target_info['build_tag']
         archlist = self.getArchList(build_tag)
 
-        data, expected_nvr = self.checkLabels(src)
+        flatpak = opts.get('flatpak', False)
+        if flatpak:
+            module = opts.get('module', None)
+            if not module:
+                raise BuildError("Module must be specifed for a Flatpak build")
+
+            module_name, module_stream, module_version = split_module_spec(module)
+
+            data = {
+                'name': module_name,
+                'version': module_stream
```

Comment:
Nit: usually nice to use a trailing comma even for the last item, to make future diffs easier to read.